frontend/packages/component-library/package.json

```diff
@@ -268,7 +268,16 @@
       "import": "./dist/esm/typography/index.mjs",
       "require": "./dist/cjs/typography/index.js"
     },
-    "./typography/index.css": "./dist/esm/typography/index.css"
+    "./typography/index.css": "./dist/esm/typography/index.css",
```

Comment:
It's probably for more for the future - let's make declarinng exports part of component development, since component is actually not available for consumers when exports are not specified

Reply:
Is this a Stephen question?